benefits/core/models.py

```diff
@@ -14,14 +16,29 @@ class PemData(models.Model):
     """API Certificate or Key in PEM format."""
 
     id = models.AutoField(primary_key=True)
-    # The data in utf-8 encoded PEM text format
-    text = models.TextField()
     # Human description of the PEM data
     label = models.TextField()
+    # The data in utf-8 encoded PEM text format
+    text = models.TextField(null=True)
+    # Public URL hosting the utf-8 encoded PEM text
+    remote_url = models.TextField(null=True)
+    # Flag if this PemData has been loaded from remote
+    remote_loaded = models.BooleanField(default=False)
 
     def __str__(self):
         return self.label
 
+    @property
+    def data(self):
+        if self.remote_loaded:
```

Comment:
I could also see this being `if self.text:` and not having to maintain the `remote_loaded` field.

On the other hand, maybe it's better to have an explicit field? What do you think?

Reply:
True, that is much simpler!